docs/how-it-works.rst

```diff
@@ -156,7 +156,7 @@ In `Swagger <https://swagger.io/>`__ documents, HTTP headers are inferred from `
 -  If request body parameters are specified as ``"in": "formData"``, request’s ``Content-Type`` header is set to ``application/x-www-form-urlencoded``.
 
 .. note::
-   Processing ``"in": "header"`` parameters and inferring ``application/x-www-form-urlencoded`` from ``"in": "formData"`` parameters is not implemented yet (`apiaryio/fury-adapter-swagger#68 <https://github.com/apiaryio/fury-adapter-swagger/issues/68>`__, `apiaryio/fury-adapter-swagger#67 <https://github.com/apiaryio/fury-adapter-swagger/issues/67>`__).
+   Processing ``"in": "header"`` parameters and inferring ``application/x-www-form-urlencoded`` from ``"in": "formData"`` parameters is not implemented yet (:ghissue:`fury-adapter-swagger#68`, :ghissue:`fury-adapter-swagger#67`).
```

Comment:
- https://github.com/apiaryio/fury-adapter-swagger/issues/67 is closed as implemented
- https://github.com/apiaryio/fury-adapter-swagger/issues/68 is also closed, `in: header is also implemented except for array headers.

I think this comment can either be removed to altered to remove some parts of it.